packages/flutter_dynamic_forms_components/lib/src/components/date/reactive_date_renderer.dart

```diff
@@ -33,20 +32,20 @@ class ReactiveDateRenderer extends FormElementRenderer<model.Date> {
                 final DateTime picked = await showDatePicker(
                   context: context,
                   firstDate: element.firstDate != null
-                      ? DateTime(element.firstDate)
-                      : DateTime(1900),
+                      ? element.firstDate
+                      : DateTime(1979 - 01 - 01),
```

Comment:
You probably meant to put comma there:  `DateTime(1979, 1, 1)`
Same with last date.

Reply:
yes you are right. i changed it.